Add notice for Field Conditions bug (#19805)
directus/directus: .changeset/fast-ways-approve.md

"@directus/app": patch
---

::: notice

> Due to a bug introduced in v10.6.2, Field Conditions were saved incorrectly. This has been corrected and works again as expected.

Author: ```suggestion
Due to a bug introduced in v10.6.2, Field Conditions were saved incorrectly. This has been fixed and works again as expected.
```

> Due to a bug introduced in v10.6.2, Field Conditions were applied incorrectly. This has been corrected and works again as expected.

Author: Suggestion from @br41nslug ❤️ 
```suggestion
Due to a bug introduced in v10.6.2, Field Conditions were saved incorrectly. This has been corrected and works again as expected.
```

> If you have edited Field Conditions under this version, please refer to https://github.com/directus/directus/issues/19757#issuecomment-1734162756 for instructions to fix the Conditions.

Author: ```suggestion
However, if Field Conditions have been edited under this version, they need to be corrected. Please refer to https://github.com/directus/directus/issues/19757#issuecomment-1734162756 for instructions on how to do so.
```

> However, if Field Conditions have been edited under this version, they need to be corrected. Please refer to https://github.com/directus/directus/issues/19757#issuecomment-1734162756 for instructions on how to do so.

Author: A.k.a the last change 😄 
```suggestion
However, if Field Conditions have been edited under this version, they need to be corrected. Please refer to https://github.com/directus/directus/issues/19757#issuecomment-1734162756 for instructions.
```
